pkgs/top-level/all-packages.nix

```diff
@@ -26551,4 +26551,5 @@ in
 
   btcdeb = callPackage ../applications/blockchains/btcdeb {};
 
+  x3270 = callPackage ../applications/misc/x3270 { };
```

Comment:
I know the current state of all-packages is in disarray, but please attempt to sort this with other entries :)